src/test/java/in/automateORM/sideMenuTest.java

```diff
@@ -0,0 +1,61 @@
+package in.automateORM;
+
+import java.util.concurrent.TimeUnit;
+
+import org.junit.Assert;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.edge.EdgeDriver;
+import org.testng.annotations.AfterClass;
+import org.testng.annotations.BeforeClass;
+import org.testng.annotations.Test;
+
+import io.github.bonigarcia.wdm.WebDriverManager;
+
+public class sideMenuTest {
+	
+	private WebDriver driver;
+	loginPageTest login = new loginPageTest();
+	
+	@BeforeClass
+	public void setUp() {
+		WebDriverManager.edgedriver().setup();
+		driver = new EdgeDriver();
+		driver.get("https://opensource-demo.orangehrmlive.com/");
+		driver.manage().window().maximize();
+		driver.manage().timeouts().implicitlyWait(10, TimeUnit.SECONDS);
+	}
+	
+	@Test(priority=1)
+	public void sideMenuOpen() {
+		
+		login.performLogin("Admin", "admin123", driver);
+		WebElement openSideMenu =  driver.findElement(By.xpath("//*[@id=\"app\"]/div[1]/div[1]/aside/nav/div[2]/div/div/button/i"));
+		openSideMenu.click();
+		openSideMenu.click();
```

Comment:
logic for two times click ?

Reply:
This method is to test, weather the side menu bar is opening and closing, 
have updated the function.